src/test/ui/generic-associated-types/mismatched-where-clause-regions.stderr

```diff
@@ -0,0 +1,42 @@
+error[E0276]: impl has stricter requirements than trait
+  --> $DIR/mismatched-where-clause-regions.rs:8:38
+   |
+LL |     type T<'a1, 'b1>
+   |     ---------------- definition of `T` from trait
+...
+LL |     type T<'a2, 'b2> = () where 'b2: 'a2;
+   |                                      ^^^ impl has extra requirement `'b2: 'a2`
+   |
+help: copy the `where` clause predicates from the trait
+   |
+LL |     type T<'a2, 'b2> = () where 'a2: 'b2;
```

Comment:
This suggestion is currently duplicated, but will not be after #106129 lands -- after that, it should only have one copy of this suggestion :)